### DISCUSSION TITLE
PHP error i need help with

### DISCUSSION THREAD

**Discussion by Tamemroofa** [community/community#121628] [Programming Help]
### Select Topic Area

Question

### Body

Hi guys, so recently i remade a website template, and in that template, i have a forum so that people may reach out to me, but everytime i try it i get a "Error: 405 https://roofa.co/forms/quote.php" error, can someone help

**Comment by itzzjb** [comment]
Check your server logs for any information about the error. It might provide more insights into what's causing the 405 error.

**Comment by lennis-dev** [comment]
I believe you used GitHub pages for your project, but PHP is its server-side programming language, which needs to be executed on the server. I think Github Pages does not support this.

**Reply by dane** [reply]
@Tamemroofa, @lennis-dev is right. You're serving [this repo](https://github.com/Tamemroofa/Roofa.co/) as your website, but GitHub Pages only serves static HTML. Your quote form could be page with an `iframe` that renders a [Wufoo form](https://www.wufoo.com/gallery/templates/forms/). That's how Jupiter Broadcasting does it. This is how [their contact page](https://www.jupiterbroadcasting.com/contact/) looks and this is how [they embed an `iframe`](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/blob/main/themes/jb/layouts/contact/single.html#L9).
